platform.md

```diff
@@ -32,13 +32,13 @@ Examples of a platform might include:
       - [Rebase](#rebase)
       - [Launch](#launch)
     - [Usage](#usage)
-      - [`detector`](#detector)
-        - [Inputs](#inputs)
-        - [Outputs](#outputs)
       - [`analyzer`](#analyzer)
         - [Inputs](#inputs-1)
         - [Outputs](#outputs-1)
         - [Layer analysis](#layer-analysis)
```

Comment:
I assume we need to move this into the restorer section now?

Reply:
Or just roll it all under `Layer Restoration`(currently `Layer restoration` but it really should be capitalized)

Reply:
I rolled it under Layer Restoration